mint-signup-form.html

```diff
@@ -0,0 +1,94 @@
+                </div>
+                <div>
+                    <img src="images/blue.png" alt="Mint-Logo"/><span class="logos-text">mint</span>
+                    <img src="images/green.png" alt="Quickbooks-Logo"/><span class="logos-text">quickbooks</span>
+                    <img src="images/red.png" alt="Turtotax-Logo"/><span class="logos-text">turbotax</span>
+                </div>
+            </div>
+
+            <div class="create-account">
+            
+                <div>
+                    <h2>Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint. <a>Learn more </a></p>
+                </div>
+                <form  method="post" id="signup-form">
+                    <fieldset>
+                        <label>Email address</label>
+                        <input type="email" id="email">
+                    </fieldset>
+
+                    <fieldset>
+                        <label>Phone (recommended)</label>
+                        <input type="tel" id="phone-number">
+                        <p id="standard-call-text">Standard call, messaging or data rates may apply.</p>
+                    </fieldset>
+                    
+                    <fieldset id="pass">
+                        <label>Password</label>
+                        <input type="password" id="password">
+                        <i class="fas fa-lock-open"></i>
+                    </fieldset>
+
+                    <div>
+                        <button class="btn" type="submit" form="signup-form"><i class="fa fa-lock"></i> Create Account</button>
+                    </div>
+
+                    <p id="create-account">
+                        By clicking Create account, you agree to our<br><a href="https://www.mint.com/terms" target="_blank">Terms</a> and have read and acknowledge our<a  href="https://security.intuit.com/privacy" target="_blank">Privacy Statement</a>.
+                    </p>
+                    <div id="form-Invisible-reCAPTCHA-p">
+                         <p >Invisible reCAPTCHA by Google <a href="https://www.google.com/intl/en/policies/privacy/">Privacy Policy </a>and<a href="https://policies.google.com/terms?hl=en"> Terms of Use.</a></p>
+                    </div>
+                </form>
+            </div>
+        </main>
+
+        <footer>
+            <div id="footer-links">
+                <a href="#">Legal</a>
+                <a href="#">Privacy</a>
+                <a href="#">Security</a>
+            </div>
+            <div id="copy-right">
+                <p>© 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+                Terms and conditions, features, support, pricing, and service options subject to change without notice.
+                </p>
+            </div>
+            <p>
+                <a href="http://jigsaw.w3.org/css-validator/check/referer">
```

Comment:
# [REQUIRED] - REMOVE THE VALIDATOR TAG
 This section is not required, Kindly remove it.
```suggestion
                <a href="http://jigsaw.w3.org/css-validator/check/referer">
```
---
This is what it is doing to your page: 👇 
 
![w3c](https://user-images.githubusercontent.com/22345961/78132176-eeaa3c00-7424-11ea-95a4-fed4051dcf33.png)

Reply:
Done